instance/models/instance.py

```diff
@@ -204,7 +204,7 @@ class GitHubInstanceMixin(VersionControlInstanceMixin):
     """
     github_organization_name = models.CharField(max_length=200, db_index=True)
     github_repository_name = models.CharField(max_length=200, db_index=True)
-    github_pr_number = models.IntegerField(blank=True, null=True)
+    github_pr_url = models.CharField(max_length=200, blank=True, default='')
```

Comment:
@Kelketek I thought that the empty string was the default for string-based fields? From a quick look at the code, https://github.com/django/django/blob/stable/1.8.x/django/db/models/fields/__init__.py#L796-L799 seem to show this behavior - `empty_strings_allowed` is the default value for a `Field` https://github.com/django/django/blob/stable/1.8.x/django/db/models/fields/__init__.py#L97 and doesn't seem to be redefined for `CharField` or `URLField`. Did you get a validation error when trying to save an `Instance` object without `default=''`?